utility/flight_stow.py

```diff
@@ -14,14 +14,37 @@ def startAnts(ants, dry_run=False):
         except:
             user_logger.warn("not all antennas were in 'STOP' mode.")
 
-def point_ants(ants, dry_run=False):
+def moveRI(ants, ridx_pos, dry_run=False):
+    if ridx_pos not in ['u', 'l', 's', 'x']:
+        raise ValueError("indexer position {} does not exist. active bands are 'u', 'l', 's', and 'x'".format(ridx_pos))
+    user_logger.info("Moving receiver indexers to '{}' position".format(ridx_pos))
+    ants.set_sampling_strategy("ap.ridx-brakes-released","period 0.5")
+    ants.req.ap_set_indexer_position(ridx_pos)
+    time.sleep(10)
+    if not dry_run:
+        try:
+    	    # Wait for indexer brakes to engage again
+            ants.wait('ap.ridx-brakes-released', False, timeout=60)
```

Comment:
Looks like an indentation error here.  Also check lines 32, 35 and 37.  It may be that the file has a mixture of tabs and spaces, in which case please change all the tabs to spaces.